src/components/AppHeader.vue

```diff
@@ -139,7 +139,7 @@ export default defineComponent({
 		const walletStore = WalletStore();
 		return {
 			EndPoint,
-			daoID: daoStore.dao_id,
+			daoID: daoStore.searchDaoId,
```

Comment:
There was a bug in this. Whenever you change the input in the search field, the other tabs gets active. Now they will be active only when dao is selected. I had to add `searchDaoId` because input change needs to be reactive.

Reply:
Since we will be storing new app (if found) in same state, then I would suggest to use one state.